src/style.css

```diff
@@ -1 +1,116 @@
-/* add styles here */
+:root {
+  --custom-color: rgba(61, 78, 97, 1);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  color: var(--custom-color);
+  font-size: 16px;
+}
+
+.search-form--large {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.search-form__label--large {
+  background-size: 19px 19px;
+  background-position: 26px center;
+}
+
+.search-form__input {
+  width: 100%;
+  height: 100%;
+
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  outline: none;
+
+  font-weight: 300;
+}
+
+.search-form__input::placeholder {
+  color: var(--custom-color);
+}
+
+.search-form__input--small:not(
+    .search-form__input--small:focus,
+    .search-form__input--small:hover
+  )::placeholder {
```

Comment:
nice!

Reply:
Hey, @eduvinagre I think the point here is that your search bar is not using the Avenir-Heavy font; you need to link it to your .ttf file using `@font-face`. Here you can see since you don't define Avenir-Heavy the site defaults to sans-serif:

Your search-bar with Avenir-Heavy:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/52002856/9b9ea72d-3dee-4951-8ffb-b92f13dccb4b)

Without it:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/52002856/c1e22eb8-66cd-48c3-b117-9bb2403e8319)

And only with Avenir-Heavy:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/52002856/8323249d-568d-49f1-b5c7-b4ff292eb23e)

Avenir-Heavy does not exist in html by default, that is why you need to use `@font-face` to import it